src/Rules/Data.hs

```diff
@@ -28,6 +28,10 @@ buildPackageData context@Context {..} = do
         -- GhcCabal may run the configure script, so we depend on it.
         whenM (doesFileExist $ configure <.> "ac") $ need [configure]
 
+        -- XXX (izgzhen): rebuild @package-data.mk@ would need @include/settings@ etc.
```

Comment:
I don't understand this comment. Please clarify. Also what is `XXX` -- a note, a to-do, something else?